modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc

```diff
@@ -3,49 +3,62 @@
 :parent-context-of-creating-a-workspace-by-importing-the-source-code-of-a-project: {context}
 
 [id="creating-a-workspace-by-importing-the-source-code-of-a-project_{context}"]
-= Creating a workspace by importing the source code of a project
```

Comment:
This heading is incorrect because this file contains four sections of which three describe importing the source code into an already created workspace = incompatible with this heading. In the one section where the source code is imported at workspace creation, this heading is still incorrect because the act of importing the source code does not create the workspace.

Reply:
Robert, your suggestion is the best in my opinion:
**Importing the source code of a project into a workspace** covers all four sub-sections.
Should I make this heading change then?

Reply:
`Importing the source code of a project into a workspace` sounds like the heading of the week :) Go for it

Reply:
Done.